scripts/codex/git-safe.ps1

```
@@ -0,0 +1,133 @@
+}
+
+function Invoke-Git {
+  param([string[]]$GitArgs)
+
+  & git @GitArgs
+  if ($LASTEXITCODE -ne 0) {
+    throw "git command failed: git $($GitArgs -join ' ')"
+  }
+}
+
+function Assert-ExplicitPaths {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Assert-ExplicitPaths' uses a plural noun. A singular noun should be used instead.

Suggested fix: Singularized correction of 'Assert-ExplicitPaths'

(PSUseSingularNouns)